src/Widgets/HeaderBar.vala

```diff
@@ -90,6 +91,14 @@ public class Camera.Widgets.HeaderBar : Gtk.HeaderBar {
         mode_switch = new Granite.ModeSwitch.from_icon_name (PHOTO_ICON_SYMBOLIC, VIDEO_ICON_SYMBOLIC);
         mode_switch.valign = Gtk.Align.CENTER;
 
+        var flip_label = new Gtk.Label ("Horizontal flip") {
```

Comment:
I think this is simpler/more common:

```suggestion
        var flip_label = new Gtk.Label ("Mirror") {
```

Reply:
Also make the string translatable: `(_("Mirror"))`

Reply:
```suggestion
        var flip_label = new Gtk.Label (_("Mirror")) {
```